app/assets/stylesheets/application.scss

```
@@ -99,7 +99,15 @@ form{
 }
 
 #todo_title {
-  color: white;
+  color: #F70A7C;
+  font-weight: 700;
   float: left;
-  font-size: 28px
+  font-size: 28px;
+  font-family: "open sans";
```

Same here:

```suggestion
  font-family: $font-open-sans;
```

(this files defines the variables you can use https://github.com/ombulabs/styleguide/blob/gh-pages/vendor/assets/stylesheets/ombulabs/styleguide/1-utilities/_variables.scss)